2023/spring/Scheffler-MW@0230PM/waiyego-maina.md

```diff
@@ -0,0 +1,8 @@
+Hi! My name is Waiyego.
+
+This is a list of things that have recently brought me joy:
+
+- My skateboard ~~even though it was confiscated at the Buenos Aires airport~~-
+- **Cooking** with my girlfriend
```

Comment:
Perhaps use triple (***) for the word to be bolder without the stars being shown